docs/references/api.rst

```diff
@@ -629,6 +631,524 @@ like ``systemctl start ubuntu-advantage.service``.
+
+      - Raised exceptions:
+
+        - No exceptions raised by this endpoint.   
+
+   .. tab-item:: CLI interaction
+      :sync: CLI
+
+      - Calling from the CLI:
+
+        .. code-block:: bash
+
+           pro api u.pro.security.fix.usn.execute.v1 --data '{"usns": ["USN-1234-1", "USN-1235-1"]}'
+
+      - Expected attributes in JSON structure:
+
+        .. code-block:: json
+
+           {
+              "usns_data": {
+                  "status": "fixed",
+                  "usns": [
+                    {
+                        "target_usn": {
+                            "title": "CVE-1234-56789",
+                            "status": "fixed",
+                            "upgraded_packages": {
+                                "name": "pkg1",
+                                "version": "1.1",
+                                "pocket": "standard-updates"
+                            },
+                            "error": null
+                        },
+                        "related_usns": []
+                    }
+                  ]
+              }
+           }
+
+   .. tab-item:: Explanation
+      :sync: explanation
+
+      When using the USN endpoint, the expected output is as follows:
+
+      .. code-block:: json
+
+          {
+            "usns_data": {
+                "status": "fixed",
+                "usns": [
+                  {
+                      "target_usn": {
+                          "title": "CVE-1234-56789",
+                          "status": "fixed",
+                          "upgraded_packages": {
+                              "name": "pkg1",
+                              "version": "1.1",
+                              "pocket": "standard-updates"
+                          },
+                          "error": null
+                      },
+                      "related_usns": []
+                  }
+                ]
+            }
+          }
+
+      From this output, we can see that the **usns_data** object contains two attributes:
+
+      * **usns**: A list of USN objects detailing what happened during the fix operation.
+      * **status**: The status of the fix operation considering **all** USNs.
+                    This means that if one USN cannot be fixed, this field will reflect that.
+                    Note that related USNs don't interfere with this attribute, meaning
+                    that a related USN can fail to be fixed without modyfing the **status**
+                    value.
+
+      Each **usn** object contains a reference for both **target_usn** and **related_usns**.
+      The target is the USN requested to be fixed by the user, while related USNs are USNs
+      that are related to the main USN and an attempt to fix them will be performed by the
+      endpoint too. To better understand that distinction, please refer to 
+      [our explanation of CVEs and USNs](cves_and_usns_explained.md).
```

Comment:
This link doesn't seem to be working. I think probably because the cves page is now in a different directory.